src/app/util/util.c

```diff
@@ -1416,3 +1416,13 @@ EmberStatus emberAfDeactivateServerTick(uint8_t endpoint, EmberAfClusterId clust
 {
     return EMBER_SUCCESS; // Stub for now.
 }
+
```

Comment:
Stub implementation tracked here
Implement stubs in zcl reporting cluster #2470